manifests/istio-control/istio-autoinject/files/injection-template.yaml

```diff
@@ -44,6 +46,7 @@ template: |
     resources: {}
   {{- end }}
     securityContext:
+  {{- if not .Values.istio_cni.enabled }}
```

Comment:
Revert this change. All securityContext fields must be set in every case. Cf. https://github.com/istio/istio/issues/17318.